tasktiger/task.py

```diff
@@ -397,6 +452,16 @@ def tasks_from_queue(self, tiger, queue, state, skip=0, limit=1000,
 
         return n, tasks
 
+    @classmethod
+    def task_count_from_queue(self, tiger, queue, state):
+        """
+        Returns the number of tasks in a given queue and task state.
+        """
```

Comment:
Implementation details aside, I'm still unsure on whether this could be solved by just using `tasks_from_queue(limit=0)[0]`. I'm aware that currently passing a 0 limit to `tasks_from_queue` returns all the tasks but this behavior is not documented, and a more Pythonic way would be to say `limit=None` if we actually wanted all the tasks. We should at least define how `tasks_from_queue` should behave before we decide to add a new function here. @jkemp101 curious if you have any thoughts on this?